tutorials/Pattern Recognition.ipynb

```diff
@@ -0,0 +1,739 @@
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# What we will do\n",
+    "\n",
+    "1. [Read item data from Clarify](#read)\n",
+    "2. [Apply a Pattern Recognition Algorithm](#apply)\n",
+    "3. [Create Patterns](#create)\n",
+    "4. [Write signals to Clarify](#write)\n",
+    "3. [Visualise the results in Clarify](#visualise)\n",
+    "\n",
+    "---\n",
+    "Other resources:\n",
+    "* [Basic tutorial on using Python with Clarify](https://github.com/searis/data-science-tutorials/blob/update_ui/tutorials/Introduction.ipynb)\n",
+    "* [API reference](https://docs.clarify.io/reference/http)\n",
+    "* [SDK documentation](https://searis.github.io/pyclarify/)\n",
+    "* [Pattern Recognition algorithm](https://matrixprofile.org)"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Read and plot item data\n",
+    "\n",
+    "<img src=\"https://raw.githubusercontent.com/searis/data-science-tutorials/pattern_recognition/media/pattern_recognition/readdata.png\" alt=\"clarify illustration\"  />"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Read Item data\n",
+    "\n",
+    "\n",
+    "First we need to install some packages, like PyClarify. PyClarify gives you a fast and easy way to get your data out of Clarify and write data back. This way you can analyze your data - for example apply a pattern recognition algorithm - and write back your results. \n",
+    "\n",
+    "> The specifics of how to use PyClarify is in the \"Basic tutorial on using Python with Clarify\" which you can find [here](https://github.com/searis/data-science-tutorials/blob/update_ui/tutorials/Introduction.ipynb). Therefore we will just briefly mention what the methods do, without going into details. "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!pip install pyclarify\n",
+    "!pip install numpy\n",
+    "!pip install matrixprofile\n",
+    "!pip install requests\n",
+    "!pip install plotly\n",
+    "!pip install --upgrade nbformat\n",
+    "!pip install pandas"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "import datetime\n",
+    "from pyclarify.client import APIClient\n",
+    "from pyclarify.models.data import DataFrame\n",
+    "from pyclarify.models.data import Signal\n",
+    "from pyclarify.models.requests import (\n",
+    "        ItemSelect,\n",
+    "        SelectItemsParams,\n",
+    "        SelectTimesParams,\n",
+    "        SelectSeriesParams,\n",
+    ")\n",
+    "import plotly.graph_objects as go\n",
+    "import plotly.express as px\n",
+    "import math"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "client = APIClient(\"./clarify-credentials.json\")\n",
```

Comment:
Maybe leave a comment that we will create multiple calls to clarify. (which is the reason for the for loop)